packages/itwinui-react/src/core/Buttons/Button.tsx

```diff
@@ -77,6 +84,8 @@ export const Button = React.forwardRef((props, ref) => {
       className={cx('iui-button', 'iui-field', className)}
       data-iui-variant={styleType !== 'default' ? styleType : undefined}
       data-iui-size={size}
+      data-iui-loading={loading}
```

Comment:
Would it be better if it is `{loading ? true : undefined}`? Because `loading` might be controlled by a state variable. And so `false` will be passed instead of `undefined` during the non-loading cases.

Reply:
fixed in [`677e8da`](https://github.com/iTwin/iTwinUI/pull/2058/commits/677e8da93e301c6fd651aaa03f59f466e7110063)